src/App.tsx

```diff
@@ -11,7 +11,7 @@ export function App(): JSX.Element | null {
   const [pid, setPid] = useState<string>();
 
   return (
-    <ChakraProvider>
+    <ChakraProvider portalZIndex={999}>
```

Comment:
why must portalZIndex be set?

Reply:
Portal z-index is set to 10 on default, it was showing up under elements of content

Reply:
gotcha, thanks